Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -112,7 +112,7 @@ def decompile(self, data, ttFont):
 		if ttFont.lazy is False: # Be lazy for None and True
 			self.ensureDecompiled()
 
-	def ensureDecompiled(self):
+	def ensureDecompiled(self, recurse=None):
```

Comment:
i think you want it to be False, not None (the latter value for recurse parameter is only relevant for TTFont which has the lazy attribute)

```suggestion
	def ensureDecompiled(self, recurse=False):
```

Also adding a comment as to why there's this parameter which is not even used inside this specific glyf table method would help our future selves